weblate/utils/templatetags/icons.py

```diff
@@ -38,10 +37,21 @@
 
 @register.simple_tag()
 def icon(name, fallback=None):
-    name = name or fallback
+    if fallback is not None:
+        name = fallback
+
+    if not name:
+        return ""
```

Comment:
This is a bug, it should not fail silently.

Reply:
I was thinking about raising ValueError here, this would be really a programming error and such should be visible while running tests, not just logged.